### ISSUE TITLE
[Adaptive Card iOS - Flight Update]: 'San Francisco Airport and SFO' text is not accessible through the swipe gesture. 

### ISSUE THREAD

**Issue by vagpt** [microsoft/Teams-AdaptiveCards-Mobile#29]
**Test Environment:**
Device: iPhone 11
iOS Version: 16.7.1
App Version: 1.0 (2.4.0-beta.23.10.15.1)
Screen Reader: Voice Over

**Repro Steps:**

1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Activate Voice over
4. Navigate to "V1.5" control and select it.
5. Navigate to 'Scenario' control and activate it.
6. Navigate to 'FlightUpdate.JSON' and double tap to activate it.
7. The Card will open.
8. Perform right swipe, navigate the card, and observe the issue.

**Actual Result:**
'San Francisco Airport and SFO' text is not accessible through the swipe gesture as after the hidden place on performing right swipe is landing on top of the page.

**Expected Result:**
'San Francisco Airport and SFO' text should be accessible through the swipe gesture and when the focus is on AMS on performing right swipe it should be land on the 'San Francisco Airport'.

**Note:**
This issue is working fine with Keyboard.

**User Impact:**
This will impact the screen reader user as due to this user might miss the valuable information available on the card.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/meaningful-sequence

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “

https://github.com/microsoft/AdaptiveCards-Mobile/assets/95913748/59f9de4f-8922-44e9-a80a-41794d6174c2

**Comment by GabrielMedAlv** [comment]
Tested using latest version and works as expected. Attached evidence. Used swipe for navigation.

https://github.com/microsoft/AdaptiveCards-Mobile/assets/130393016/c9aa2063-a8de-4ba0-91c7-00f5de78f535

**Comment by vagpt** [comment]
This issue is still repro on build 1.0 (2.4.0-beta.24.11.3.1) hence reactivating this issue.

**Comment by GabrielMedAlv** [comment]
Merged PR with fix.
